vispy/testing/_runners.py

```diff
@@ -190,8 +190,14 @@ def _check_line_endings():
 """
 
 
-def _examples():
-    """Run all examples and make sure they work
+def _examples(individial_file_paths):
+    """Run examples and make sure they work.
+
+    Parameters
+    ----------
+
+    individial_file_paths: str
```

Comment:
In my mind they're basically the same thing -- a list of paths. It's just that one is space-separated, and the other is a list of strings. Two advantages to re-using the name in Python: 1) it keeps the namespace tidier, and 2) automatic GC can take over the no-longer-needed name.